service/awsconfig/v7/templates/cloudformation/guest/lifecycle_hooks.go

```diff
@@ -0,0 +1,13 @@
+package guest
+
+const LifecycleHooks = `{{define "lifecycle_hooks"}}
```

Comment:
In order to inspect lifecycle states of instances we need to add lifecycle hooks. Therefore the template.

Reply:
Do I understand correctly it blocks proceeding till `CompleteLifecycleAction` call?